Lesson31/Kubra/my-app/src/components/ListItem/index.jsx

```diff
@@ -1,9 +1,11 @@
 import './styles.css';
 
-export const ListItem = ({id, title, completed, handleClick}) => 
-  <li
-      className={completed ? "completed" : null}
-      onClick={() => handleClick(id)}
-
-    >{title}</li>;
+export const ListItem = ({id, title, completed, handleClick, handleDelete}) => 
+ ( <li
```

Comment:
Your solution works but could be improved a bit but doing the following:
```
<li>
    <span className={completed ? 'completed' : null} onClick={() => handleClick(id)}>{title}</span>
		<button onClick={() => handleDelete(id)}>X</button>
	</li>
```
I add an extra span element to hold the text of the todo item and moved completed class and handle click event into it. This way, the delete button will not be checked if we click on the item.